sdk/scheduler/src/main/java/com/mesosphere/sdk/curator/CuratorUtils.java

```diff
@@ -20,14 +20,14 @@
 
     private static final String PATH_DELIM = "/";
 
-    static final int DEFAULT_CURATOR_POLL_DELAY_MS = 1000;
-    static final int DEFAULT_CURATOR_MAX_RETRIES = 3;
+    public static final int DEFAULT_CURATOR_POLL_DELAY_MS = 1000;
```

Comment:
Looks like `CuratorConfigStore` + `CuratorStateStore` were both using these to do the following:

```
new ExponentialBackoffRetry(
                CuratorUtils.DEFAULT_CURATOR_POLL_DELAY_MS,
                CuratorUtils.DEFAULT_CURATOR_MAX_RETRIES)
```

Could:
- Mark these private
- Add a static `getDefaultRetryPolicy()` function
- Switch `CuratorConfigStore`/`CuratorStateStore`/`DefaultScheduler` to call that function